test/rdoc/test_rdoc_generator_darkfish.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative 'helper'
 
-class TestRDocGeneratorDarkfish < RDoc::TestCase
+class RDocGeneratorDarkfishTest < RDoc::TestCase
```

Comment:
This class name change is... unconventional (I looked at the neighboring files). Is it intentional?

Reply:
@st0012 Thanks for providing that context. Seems like a good thing!